tests/httptoolkit/service/test_async_service.py

```diff
@@ -0,0 +1,242 @@
+import json
+from datetime import timedelta
+
+import httpx
+import pytest
+from pytest_httpx import HTTPXMock
+from typing import Any
+
+from httptoolkit import Header, AsyncHttpxService
+from httptoolkit.errors import HttpError, ServiceError
+from httptoolkit.service import AsyncService
+from httptoolkit.transport import AsyncHttpxTransport
+from tests.httptoolkit.conftest import CustomJSONEncoder
+
+
+class DummyAsyncService(AsyncService):
+    pass
+
+
+@pytest.fixture()
+def async_service() -> AsyncService:
+    return DummyAsyncService(
+        transport=AsyncHttpxTransport(base_url="https://kontur.ru", proxies={"http://": "http://10.10.1.10:3128"}),
```

Comment:
It's better to replace real domain examples with "example.com" or similar.

Reply:
fixed